src/pages/mock-openings.js

```diff
@@ -0,0 +1,76 @@
+import React, { Fragment } from 'react';
+
+import Button from '../components/Button';
+import Footer from '../components/Footer';
+import MainNav from '../components/MainNav';
+import SEO from '../components/SEO';
+
+const MockOpenings = () => (
+  <Fragment>
+    <SEO title="Mock Openings" />
+    <MainNav title="Mock Openings" />
+    <div className="container py-16 blog-content md:flex">
+      <div className="flex-1 md:pl-16">
+        <p>
+          In Coding Coach beside our open-source application,  we have an opportunity for all developers to try a mock job opportunity. This way developers can learn the common process of applying for a job, how to prepare C.V. and cover letters, different steps of the process, and estimate themself and their knowledge during the different rules. We are defining a company named CodingCoach CO. as our fake company.
+        </p>
+
+        <div className="bg-red-100 border border-red-400 text-red-700 px-4 py-3 rounded relative" role="alert">
```

Comment:
The colors are not working because they don't exist in our setup. Please use one of the colors available here: https://github.com/Coding-Coach/coding-coach/blob/master/tailwind.config.js#L5-L32

```
bg-danger-lighter border-danger-darker
```